config.docker.php

```diff
@@ -4,6 +4,17 @@
  *   ln -s config.docker.php config.php
  */
 
+function file_env($var, $default) {
+  $filevar = $var . '_FILE';
+  if (getenv($filevar)) {
```

Comment:
Can we add a file_exists check here?

https://www.php.net/manual/en/function.file-get-contents.php
An **E_WARNING** level error is generated if filename cannot be found, maxlength is less than zero, or if seeking to the specified offset in the stream fails.

Reply:
I added in a check using is_readable instead of file_exists. That way it will verify the permissions in addition to verifying the existence of the file. At the same time I didn't want to hide the error since that would make it difficult to diagnose why something isn't working, so I included an error_log line if the file cannot be read.